src/transformers/modeling_utils.py

```diff
@@ -786,6 +789,7 @@ def generate(
         attention_mask=None,
         decoder_start_token_id=None,
         use_cache=None,
+        **model_specific_kwargs
```

Comment:
For Reformer, the accuracy can manually be controlled by the function argument `num_hashes`. It's so specific to one model that I thought it's better to have a ``model_specific_kwargs`` argument here. @thomwolf @sshleifer

Reply:
maybe name it forward_kwargs?